index.html

```diff
@@ -4,18 +4,172 @@
+            <h3>100% Natural Ingredients</h3>
+            <p>I'm a paragraph. Click here to add your own text and edit me. Let your users get to know you.</p>
+        </div>
+
+        <div class="promo-wrapper-3">
+            <div class="promo-section-image"><img src="images/11062b_3dcda4e117574ddbb4e50268223b3f1a.svg" /></div>
+            <h3>Just Picked Freshness</h3>
+            <p>I'm a paragraph. Click here to add your own text and edit me. Let your users get to know you.</p>
+        </div>
+    </section>
+
+
+
+
+
+    <section class="background-color-2-section">
+        <div class="background-color-2-section-background"></div>
+        <div class="background-color-2-section-img"></div>
+        <h2>Boosting<br>Immunity the Way Nature Intended</h2>
+        <p>I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to
+        add your own content and make changes to the font. I’m a great place for you to tell a story and let your users know a
+        little more about you.</p>
+        <button class="button-no-hover"><a href="#">Our Vision</a></button>
+    </section>
+
+
+    <section class="background-color-1-section">
+        <div class="background-color-1-section-background"></div>
+
+        <h2>Choose Your Wellness Path</h2>
+        <div class="background-color-1-section-img-1"></div>
+        <h3 class="color-1-section-img-1-header">Pure Hydration with Сold Pressed Juices</h3>
+        <p class="color-1-section-img-1-p">I'm a paragraph. Click here to add your own text and edit me. Let your users get to know you.</p>
+
+        <div class="background-color-1-section-img-2"></div>
+        <h3 class="color-1-section-img-2-header">Soft & Creamy Organic Smoothies</h3>
+        <p class="color-1-section-img-2-p">I'm a paragraph. Click here to add your own text and edit me. Let your users get to know you.</p>
+
+        <div class="background-color-1-section-img-3"></div>
+        <h3 class="color-1-section-img-3-header">The Power of Wellness Shots</h3>
+        <p class="color-1-section-img-3-p">I'm a paragraph. Click here to add your own text and edit me. Let your users get to know you.</p>
+
+        <button class="button-no-hover"><a href="#">View All</a></button>
+    </section>
+
+
+    <section class="background-main-color-section">
+        <div class="background-main-color-section-img"></div>
+        <h2>Discover Our Special Cleanse Kit</h2>
+        <p>I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to
+        add your own content and make changes to the font. Feel free to drag and drop me anywhere you like on your page. I’m a
+        great place for you to tell a story and let your users know a little more about you.​</p>
+        <button class="button-no-hover"><a href="#">Try It</a></button>
```

Comment:
Again here if you put these 3 elements (heading, text and button) inside a div, you'll be able to place these on the grid more easily.